.circleci/config.yml

```diff
@@ -32,34 +32,35 @@ jobs:
           paths:
             - .
 
-  deploy-rinkeby:
-    executor: node/default
-    working_directory: ~/project
-    steps:
-      - attach_workspace:
-          at: ~/project
-      - azure-cli/install
-      - azure-cli/login-with-service-principal
-      - run:
-          name: Install Azure functions tools
-          command: sudo npm i -g azure-functions-core-tools@4 --unsafe-perm true
-      - run:
-          name: Deploy to Azure function App
-          command: func azure functionapp publish af-datastore-api-rinkeby-prod --typescript
-  deploy-mainnet:
-    executor: node/default
-    working_directory: ~/project
-    steps:
-      - attach_workspace:
-          at: ~/project
-      - azure-cli/install
-      - azure-cli/login-with-service-principal
-      - run:
-          name: Install Azure functions tools
-          command: sudo npm i -g azure-functions-core-tools@4 --unsafe-perm true
-      - run:
-          name: Deploy to Azure function App
-          command: func azure functionapp publish af-datastore-api-rinkeby-prod --typescript
+  # deploy-rinkeby:
```

Comment:
Why comment all of this out instead of remove it?

Reply:
Perhaps we may want it in the future 🤷